server/src/messaging/RabbitMQMessaging.js

```diff
@@ -0,0 +1,19 @@
+import { BrokerAsPromised as Broker } from 'rascal';
+import { config } from './config';
+
+const publishMessage = (async (payload, resultHandler) => {

```

Comment:
Hi @nmcharlton How do I fix the following alerts below? I really don't have a type for the `payload` or have a return type for this function.